plugin_configs/speedtest_config.py.sample

```diff
@@ -0,0 +1,46 @@
+# Number of seconds to show speedtest screen. Default is 15.
+PAUSE = 15
+
+# The download speed (in Mbps) that you are promised to be provided by your ISP.
+# Note: this is megabits per second, not megabytes per second
+PROMISED_DL_SPEED = 200
+
+# The upload speed (in Mbps) that you are promised to be provided by your ISP.
+# Note: this is megabits per second, not megabytes per second
+PROMISED_UL_SPEED = 10
+
+# If speedtest is at or below this percentage, show the dial as red.
+RED_CUTOFF = 75
+
+# If speedtest is at or below this percentage, but greater than the RED_CUTOFF,
+# show the dial as yellow. Anything above this percentage will be show as green.
+YELLOW_CUTOFF = 90
+
+# Only the last few test results will be kept, by default. They can be found at:
+# /home/pi/PiWeatherRock/speedtest/queue/
+# If set to True, old resuts will be moved to:
+# /home/pi/PiWeatherRock/speedtest/archive/
+KEEP_ALL_SPEEDTESTS = False
+
+# Show the percentage of download and upload 'promised speeds' near bottom-right
+# side of the dial.
+SHOW_SPEEDTEST_PERCENTAGE = True
+
+# Show 'Mb/s' below the number representing your download and upload speeds in then
```

Comment:
Since you commented about megabits vs megabytes above, maybe note which this is an abbreviation of.

Reply:
good idea. done